### ISSUE TITLE
Invalid literal for int() in AFNISkullStripWorkflow

### ISSUE THREAD

**Issue by jokedurnez** [poldracklab/mriqc#694]
Running mriqc_0.10.0 (latest singularity container on sherlock):

```
more crash-20180219-223807-jdurnez-skullstrip.a0-fd7decb7-03b1-4057-8627-3b078d699d4e.txt
Node: workflow_enumerator.anatMRIQCT2w.AFNISkullStripWorkflow.skullstrip
Working directory: /lscratch/jdurnez/workflow_enumerator/anatMRIQCT2w/AFNISkullStripWorkflow/_in_file_..oak..stanford..groups..russpold..data..Psychosis..0.0.2..sub-S9184VMG..anat..sub-S9184VMG_run-1_T2w.nii.
gz/skullstrip

Node inputs:

args = <undefined>
environ = {}
ignore_exception = False
in_file = /lscratch/jdurnez/workflow_enumerator/anatMRIQCT2w/AFNISkullStripWorkflow/_in_file_..oak..stanford..groups..russpold..data..Psychosis..0.0.2..sub-S9184VMG..anat..sub-S9184VMG_run-1_T2w.nii.gz/inu_n4
/sub-S9184VMG_run-1_T2w_conformed_corrected.nii.gz
num_threads = 1
out_file = <undefined>

outputtype = NIFTI_GZ
terminal_output = <undefined>

Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/multiproc.py", line 51, in run_node
    result['result'] = node.run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 407, in run
    self._run_interface()
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 517, in _run_interface
    self._result = self._run_command(execute)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 650, in _run_command
    result = self._interface.run()
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base.py", line 1062, in run
    env['DISPLAY'] = config.get_display()
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/utils/config.py", line 283, in get_display
    ndisp = int(sysdisplay.split(':')[-1])
ValueError: invalid literal for int() with base 10: '18.0'
```

**Comment by chrisfilo** [comment]
Seems to be a duplicate of https://github.com/poldracklab/mriqc/issues/672

**Comment by chrisfilo** [comment]
Also - this should be fixed in 0.10.1 since it included nipype update: https://github.com/nipy/nipype/pull/2363

**Comment by jokedurnez** [comment]
Tried 0.10.2 and problem is indeed fixed.  Thanks !
